parquet-hadoop/src/main/java/org/apache/parquet/filter2/bloomfilterlevel/BloomFilterImpl.java

```diff
@@ -119,6 +119,16 @@ public <T extends Comparable<T>> Boolean visit(Operators.GtEq<T> gtEq) {
     return BLOCK_MIGHT_MATCH;
   }
 
+  @Override
+  public <T extends Comparable<T>> Boolean visit(Operators.Contains<T> contains) {
+    return BLOCK_MIGHT_MATCH;
```

Comment:
I was testing this out with the BloomFilterVisitor and couldn't produce a test Parquet file with a populated bloom filter for a List field (I tried strings, ints, and longs as element types). I guess this is expected?

Reply:
that worked! thanks 👍